content/components/action-bar.mdx

```diff
@@ -0,0 +1,277 @@
+
+<CustomVideoPlayer
+  width="720px"
+  loop
+  src="https://user-images.githubusercontent.com/378023/188359460-bc88bac8-9c69-4aea-8ce0-bc427bedc3a3.mov"
+  poster="https://user-images.githubusercontent.com/378023/188529873-eafbad89-9eeb-4342-905d-15d89d288ffd.png"
+/>
+<Caption>Overflow button appears when not enough space and resizing the action bar updates the overflow menu.</Caption>
+
+## States
+
+### Button states
+
+Buttons in action bars are solely used for triggering actions. Consider using a [segmented control](https://primer.style/design/components/segmented-control) when a button should have a selected state.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/378023/187873600-b77626d4-a832-4297-90bc-b0cb1060b9c1.png"
+      role="presentation"
+      width="456"
+    />
+    <Caption>Buttons in action bars have a hover and pressed state. Or a focused state when using the keyboard to navigate.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/378023/187875081-42478d03-e650-414c-81c2-6d04e8e3a398.png"
+      role="presentation"
+      width="456"
+    />
+    <Caption>Don't add a selected state or any other information like a notification dot or a counter.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### Tooltips
+
+When hovering over a button for a bit, a tooltip will appear that describes the action.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/378023/188030590-f1bceb3a-184a-438d-9885-db7c9ecc8c36.png"
+      role="presentation"
+      width="456"
+    />
+    <Caption>Describe what action will be taken when clicking on the button.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/378023/188030603-4bb5c238-edf4-41ed-9113-0d59649f0d5b.png"
+      role="presentation"
+      width="456"
+    />
+    <Caption>Don't use a tooltip in action bars to convey a current state.</Caption>
+  </Dont>
+</DoDontContainer>
+
+## Options
+
+### Size
+
+Action bars can have 3 different sizes:
+
+<img
+  src="https://user-images.githubusercontent.com/378023/188034827-840e6fda-11ad-45ae-80ac-8eeae20ab618.png"
+  role="presentation"
+  width="960"
+/>
+
+- Small (`28px`)
+- Medium (`32px`) (default)
+- Large (`40px`)
+
+## Layout
+
+Action bars can be used inline next to other content or also full width taking up the entire space.
+
+<Box as="p">
+  <img
+    src="https://user-images.githubusercontent.com/378023/188048756-73d282d7-05f9-4a97-b02a-78afbb10870b.png"
+    role="presentation"
+    width="960"
+  />
+</Box>
+
+### Spacing
+
+Make sure to add extra spacing around the action bar.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/378023/188040639-ef9bf330-ff9f-4167-957c-ff8fd7ea7b8d.png"
+      role="presentation"
+      width="456"
+    />
```

Comment:
I think the example doesn't express spacing/padding around the items. What about painting that area?

<img width="456" alt="Do" src="https://user-images.githubusercontent.com/912236/188564460-097dc4e4-3f5f-4948-8ba2-56a314e598d1.png">

Reply:
> What about painting that area?

Ahh.. yes. That's better. 👍 Updated.